packages/bsp/common/usr/lib/armbian/armbian-resize-filesystem

```diff
@@ -13,208 +13,287 @@
 
 do_expand_partition()
 {
-	# check if growroot (from cloud-initramfs-growroot package) is installed.
-	# despite it's name, that package does NOT require cloud-init.
-	# if so, it means the partition with root filesystem was already resized during initramfs.
-	# in this case do nothing here, but return 0 to allow resize2fs to run (growroot does not handle that).
-	if [[ -f /usr/share/initramfs-tools/hooks/growroot ]] || [[ -f /usr/share/initramfs-tools/scripts/local-bottom/growroot ]]; then
-		echo "partition resize skipped: growroot detected."
-		return 0
+	local partdev=$1
+	echo -e "\n### [resize2fs] Trying to resize partition $partdev:\n"
+
+	# make sure the target is a partition
+	local parttype=$(lsblk -n -o TYPE $partdev | head -1)
+	if [[ "$parttype" != "part" ]]; then
+		echo -e "\n$partdev isn't a partition: $parttype" >&2
```

Comment:
Wonder if printf would be better here. Dunno.

Reply:
Cosmetic change ...